app/app.go

```diff
@@ -487,7 +486,7 @@ func NewShentuApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 		authz.NewAppModule(appCodec, app.AuthzKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry),
 		bank.NewAppModule(appCodec, app.BankKeeper, app.AccountKeeper),
 		capability.NewAppModule(appCodec, *app.CapabilityKeeper),
-		crisis.NewAppModule(&app.CrisisKeeper, skipGenesisInvariants),
+		crisis.NewAppModule(&app.CrisisKeeper, true), // skip genesis invariant check for now
```

Comment:
guess this could be left as it is? 
because crisismodule.InitGenesis will have no chance to be called in the upgradeHandler.

Reply:
Hmm that's true. I'll fix it